<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interaction</class>
 <widget class="QDialog" name="Interaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Никнейм:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okayButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled {
 background:#1A1A1A; 
color:#9B9B9B;
}</string>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled {
 background:#1A1A1A; 
color:#9B9B9B;
}</string>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
